Products/GenericSetup/utils.py

```diff
@@ -764,7 +764,10 @@ def _initProperties(self, node):
             remove_elements = []
             for sub in child.childNodes:
                 if sub.nodeName == 'element':
-                    value = sub.getAttribute('value').encode(self._encoding)
+                    value = sub.getAttribute('value')
+                    if prop_map.get('type') not in (
+                            'ulines', 'multiple selection'):
+                        value = value.encode(self._encoding)
```

Comment:
Should this include the other unicode field types?  I.e., `ustring`, `utokens`, `utext`